.gitignore

```diff
@@ -22,6 +22,7 @@ Icon
 
 # pytest
 .pytest_cache
+.cache
```

Comment:
You're using an old version of pytest. Since 3.4.0 pytest will create the cache directory in `.pytest_cache`. See #13308 :)

(I think we can still leave it there for users with old pytest versions)

Reply:
Aah, I was already noticing that dir and was like: where did it come from.

Reply:
Ah, good call! In any case I'm sure ignoring `.cache` doesn't hurt.